src/Cafe/HW/Latte/Renderer/Vulkan/VulkanRenderer.cpp

```diff
@@ -1106,7 +1108,7 @@ bool VulkanRenderer::CheckDeviceExtensionSupport(const VkPhysicalDevice device,
 	{
 		requiredExtensions.erase(extension.extensionName);
 	}
-
+	
```

Comment:
It isn't clear by the diff, but it seems that here you added a bit of unneeded whitespace

Reply:
done